test/gtest/uct/ib/test_ib.cc

```diff
@@ -445,9 +445,19 @@ class test_uct_ib_sl : public test_uct_ib_with_specific_port {
 };
 
 UCS_TEST_P(test_uct_ib_sl, check_ib_sl_config) {
+    const char *avail_sls_str = getenv("GTEST_AVAIL_SL");
+    unsigned short avail_sls;
+    uint8_t sl;
+
+    if (avail_sls_str == NULL) {
+        avail_sls = 0xffff;
+    } else {
+        avail_sls = (unsigned short)strtoul(avail_sls_str, NULL, 16);
+    }
+
     // go over all SLs, check UCTs could be initialized on a specific SL
     // and able to send/recv traffic
-    for (uint8_t sl = 0; sl < UCT_IB_SL_NUM; ++sl)  {
+    for (sl = 0; (sl < UCT_IB_SL_NUM) && ((UCS_BIT(sl) & avail_sls) != 0); ++sl)  {
```

Comment:
Nope. The first unavail SL will kick out from the loop.

Reply:
Right, I am just fixing it